_dashboards/admin-ui-index/index-management.md

```diff
@@ -0,0 +1,242 @@
+
+* Select the blue `Create rollup job` button on the `Rollup Jobs` page under `Index Management`.
+* Set the name, source index, and target index.
+* Select `Next`.
+* Set the timestamp field and interval type.
+* Optionally you can set additional aggregations and additional metrics.
+* Select `Next`.
+* Under `Schedule`, check or uncheck `Enable job by default`.
+* Set the Continuous, execution frequency, rollup interval and page per execution settings.
+* Additionally, you can set an execution delay.
+* Select `Next`.
+* Review the settings for the rollup job and select `Create`.
+
+You can also disable and enable rollup jobs by selecting the corresponding buttons on the `Rollup Jobs` page.
+
+(Screenshots)
+
+## Transform Jobs
+
+Using the admin UI in OpenSearch Dashboards, you can create, start, stop, and complete operations with [transform]({{site.url}}{{site.baseurl}}/im-plugin/index-transforms/transforms-apis/) jobs.
+
+To create a transform job:
+
+* Select the blue `Create transform job` button on the `Transform Jobs` page under `Index Management`.
+* Set the name, source index, and target index.
+* Select `Next`.
+* Select the fields to transform. From the table above, select a field you want to transform by clicking `+` next to the field name.
+* Select `Next`.
+* Check or uncheck `Job enabled by default`.
+* Set wheter the schedule is continuous and the transform execution interval.
+* Optionally you can set pages per execution under the `Advanced` drop-down.
+* Select `Next`.
+* Review the settings for the rollup job and select `Create`.
+
+You can also disable and enable rollup jobs by selecting the corresponding buttons on the `Transform Jobs` page.
+
+(Screenshots)
+
+## Long running operation status check 
+
+Certain index operations take time to complete (usually more than 30 secs, but could take tens of minutes or hours). This is tracked on the index status on indices page.
+
+For reindex, shrink, and split operations as they are one-time operations and not recursive, you can check the new index status for the progress of these operations.
+
+## Security Integration
+
+  Permission control is managed with existing [permissions]({{site.url}}{{site.baseurl}}/security-plugin/access-control/permissions/) or action groups that are enforced at the API level. There is currently no UI level permission control. Whoever can access the ISM plugin is able to view new pages, but whether they can make changes depends on whether they have permission to run the related APIs.
+
+## Error Handling
+
+Similar to API calls, if the operation fails immediately, you will be notified with an error message. However, if it is a long running operation, you will be notified of the failure at the time of failure, or you can check the index status on the `Indices` page.
+
+(Screenshot)
+
+## Notifications
+
+The admin UI uses the notification plugin to inform you of the completion of long running operations such as split, shrink, and reindex.
```

Comment:
For 2.5 release, we have not integrated with notification plugin yet. But there will have a toast in UI telling the result of those operation result.

Reply:
alright sounds good. I'll get this changed.